src/modules/app/route-configs/ExtrasRouteConfig.js

```diff
@@ -47,12 +48,23 @@ class ExtrasRouteConfig implements RouteConfig<ExtrasRouteParamsType, RequiredPa
 
   getRoutePath = ({ city, language }: ExtrasRouteParamsType): string => `/${city}/${language}/extras`
 
-  getRequiredPayloads = (payloads: AllPayloadsType): RequiredPayloadsType => ({ extras: payloads.extrasPayload })
+  getRequiredPayloads = (payloads: AllPayloadsType): RequiredPayloadsType => ({
+    cities: payloads.citiesPayload,
+    extras: payloads.extrasPayload
+  })
 
   getLanguageChangePath = ({ location, language }) =>
     this.getRoutePath({ city: location.payload.city, language })
 
-  getPageTitle = ({ t, cityName }) => cityName ? `${t('pageTitles.extras')} - ${cityName}` : null
+  getPageTitle = ({ t, payloads, location }) => {
+    const cityModel = payloads.cities.data
+      ? payloads.cities.data.find(cityModel => cityModel.code === location.payload.city)
+      : null
+    if (!cityModel || !cityModel.extrasEnabled) {
+      return null
```

Comment:
This means the page name defaults to `Integreat`, right?

Reply:
Yes